chukwuemekachm/utility_manager_web
# Prof: User update profile

#### What does this PR achieve?
- updates dateHelpers to contain functions
- updates UpdateProfile page
- breaks errors in  Input component to a new InputErrors component
- adds an ImagePicker component for easily picking images
- adds docs for all components added
- makes API calls to update profile


#### Manual Testing Steps
- navigate /dashboard/profile
- you should be able to update your profile
- submit button should be disabled by default until you change at least one field

#### Screenshots:
<img width="345" alt="Screenshot 2020-02-09 at 1 05 37 PM" src="https://user-images.githubusercontent.com/38565349/74102265-39ed5000-4b42-11ea-840c-49223d28d903.png">
<img width="354" alt="Screenshot 2020-02-09 at 1 49 11 PM" src="https://user-images.githubusercontent.com/38565349/74102334-fcd58d80-4b42-11ea-96cc-d5d825da8b00.png">

#### Relevant Trello Tickets
- [Update profile](https://trello.com/c/ilJbUWsc/52-d-2-user-should-be-able-to-update-their-profiles-from-their-dashboard
